src/Symfony/Component/HttpKernel/HttpCache/ResponseCacheStrategy.php

```diff
@@ -199,9 +200,18 @@ private function willMakeFinalResponseUncacheable(Response $response): bool
      * If the value is lower than the currently stored value, we update the value, to keep a rolling
      * minimal value of each instruction. If the value is NULL, the directive will not be set on the final response.
```

Comment:
that doc line needs to be updated for $isHeuristicallyCacheable

Reply:
I have improved the docblock to explain what is going on